source/developers-guide/rest-api/api-resource-article/index.md

```diff
@@ -213,12 +213,26 @@ The article(s) to delete can be defined using the following syntax:
 
 ## DELETE (Stack)
 
-In order to delete more than one article at once, it's possible to provide an array of ids to the REST API.
-Simply pass the array of article ids to the following URL (example)
+In order to delete more than one article at once, it's possible to provide an array of objects with ids or article numbers to the REST API.
```

Comment:
I know it is the "article" resource, but it hurts my eyes, when I'm reading of `articles` :smile: 
I think we should use the correct translation `product` if we are talking about non-code-related stuff

what do you think about that @soebbing ?

Reply:
I agree, but we should do so completely. Changing it only here would be suprising.

Reply:
I made a PR which changes "article" to "product" where i thought it would fit

Reply:
@soebbing I merged your PR.

@Pmuckel please rebase your branch :blush:

Reply:
@mitelg done